### PR TITLE
Update CI to deploy to staging

### PR DESCRIPTION
**Pull request by akuanti**
Commits to master deploy to the live site, while commits to develop now deploy to the staging site.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -99,7 +99,8 @@ jobs:
            . venv/bin/activate
            sh ./scripts/publish-api.sh
 
-  dev-deploy-api:
+  # NOTE: namespace still needs to be "development"
```
Comment (kangarang) [inline .circleci/config.yml]: is this temporary? as in, we shouldn't change the namespace until we merge the updated abis into `master` b/c of env vars?

File: .circleci/config.yml [hunk 2]
```diff
@@ -373,47 +439,73 @@ workflows:
...
-      - dev-deploy-frontend:
+      - staging-deploy-frontend:
           requires:
             - publish-frontend-image
           filters:
             branches:
               only:
                 - develop
+      # deploy to live on push to master
+    #   - approve:
```
Comment (kangarang) [inline .circleci/config.yml]: what does this code do if uncommented?

File: .circleci/config.yml (hunk 1 above)
Comment (akuanti) [reply .circleci/config.yml]: This refers to the Kubernetes namespace where the config and the containers are running. We have to create a new one and install the appropriate configuration there before we can change this.

File: .circleci/config.yml (hunk 2 above)
Comment (akuanti) [reply .circleci/config.yml]: It would require a couple other changes, but it would only deploy with a manual approval in CircleCI.

**Pull request merged**